src/raw/aws/fargate_raw_metrics.go

```diff
@@ -36,7 +36,6 @@ func fargateRawMetrics(fargateStats FargateStats) map[string]*raw.Metrics {
 				ThrottledPeriods:  stats.CPUStats.ThrottlingData.ThrottledPeriods,
 				ThrottledTimeNS:   stats.CPUStats.ThrottlingData.ThrottledTime,
 				SystemUsage:       stats.CPUStats.SystemUsage,
-				OnlineCPUs:        uint(stats.CPUStats.OnlineCPUs),
```

Comment:
We were not using this value before, so after the needed changes in [biz/metrics.go](https://github.com/newrelic/nri-docker/pull/105/files#diff-f904c524386409b976146175f99fcb7ed6b5d7e450a5de0e3cc361cbc6e0ac88R166), the way I found to keep the previous behaviour is to stop defining `OnlineCPUs` value here.

I created the issue #104 to review if we would rather prefer to honor the `OnlineCPUs` value from the API in fargate.